gcp.pkr.hcl

```diff
@@ -88,6 +88,10 @@ build {
       "shared/scripts/gvisor.sh",
       "shared/scripts/apt-install-jq.sh",
     ]
+
+    env = {
+      DEBIAN_FRONTEND = "noninteractive"
```

Comment:
I want to get rid of red warnings 🙂 
<img width="1135" alt="image" src="https://github.com/user-attachments/assets/219776c4-8f93-461a-a9af-0e74369b1220">